Clean nsfw

Zalobomb opened the pull request:
Minor code cleanup and regex patching

Zalobomb committed:
Cleaned up some of the NSFW duplication

Zalobomb committed:
cleaned more code, introduced method to handle link conversion, homogenified all links, regardless of v.redd.it or regular links

Zalobomb committed:
Stage three of cleanup, bot now works with considerably less code, corrected call mistakes of previous push

Zalobomb committed:
Re-added thankreply import

Zalobomb committed:
Fixed regex in initial string matching on line 107

Zalobomb committed:
Re-added thank replies, made note of directory

Tabrnicus commented on chetbot.py:
```diff
@@ -51,79 +48,42 @@ async def on_message(message):
+
+        if json_data[0]['data']['children'][0]['data']['over_18']:
+            raw_video_name = "SPOLIER_" + raw_video_name
+            compressed_video_name = "SPOILER_" + compressed_video_name
```
imagine failing spelling :kappa:

jtesiorna committed:
Fixed compress_video function
-Fixed the input variables for the compress_video function so that it actually points to the video we want to compress instead of it being 'input.mp4' and 'output.mp4'
